src/test/java/org/assertj/core/api/iterable/IterableAssert_containsAnyElementsOf_Test.java

```diff
@@ -49,8 +49,9 @@ protected void verify_internal_effects() {
   public void should_compile_as_containsAnyElementsOf_declares_bounded_wildcard_parameter() {
     // GIVEN
     Map<String, String> map = newHashMap("some_key", "some_value");
+    Iterable<Map<String, String>> iterable = list(map);
     // THEN
-    assertThat(map).extracting("some_key").containsAnyElementsOf(list("some_value", "some_other_value"));
+    assertThat(iterable).extracting("some_key").containsAnyElementsOf(list("some_value", "some_other_value"));
```

Comment:
add a new test checking `extracting` with multiple arguments since it is now missing

Reply:
I was not sure how to update this test because it is targeting an `Iterable` and from my point of view the `Map` was not really relevant here (also because the `extracting` of `AbstractIterableAssert` is never reached in case of maps). Since I was changing the `extracting` in `AbstractMapAssert` this test was also affected, therefore my quick fix was to wrap the `Map`.

Instead of adding another test here, I would suggest to remove the `Map` instance and to use any `Iterable` instead. What do you think?

Reply:
I agree, this test actually was to make sure that `containsAnyElementsOf` could accept more than a `List<Object>`